ndk_compat/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+
+if(TRUE OR ANDROID)
```

Comment:
not sure is relevant since you already check in in Root CMakeLists.txt
or in root inconditionnaly `add_subdirectory()` and check here..